src/elm/Session/LoggedIn.elm

```diff
@@ -125,6 +126,7 @@ type alias Model =
     , showAuthModal : Bool
     , auth : Auth.Model
     , balances : List Balance
+    , feedback : Maybe Feedback.Model
```

Comment:
I like this, but `Maybe` isn't good. We can achieve the same result but with lot of more semantics by introducing a type:

```elm
type Feedback = 
     Dismissed
     | Enabled Feedback.Model
```

Same result but now can do cases without having to handle `maybe`. Not to say that it isn't useful, but it will make the debugger easier to read in the future since we will have some error like "you didn't implement all possible values for `Feedback`" instead of "the function is expecting Maybe Feedback.Model, but the provided value is Feedback.Model"

It will also avoid putting `Just` everywhere too

Reply:
If you want to go deeper into types you may take a look at Theorem proving languages like Coq or Idris. Their compiler and type notion is much improved.

I also recommend this exercise in Haskell, that is not specific to the discussion on `Maybe vs types` but does highlight how we can use the compiler to help us keep context in our application: 

https://reasonablypolymorphic.com/blog/typeholes/index.html